src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -8881,8 +8882,8 @@ namespace HeatBalanceManager {
                                          state.dataIPShortCut->rNumericArgs(10)));
             }
 
-            if (state.dataHeatBal->ComplexShade(Loop).LayerType == csVenetianHorizontal ||
-                state.dataHeatBal->ComplexShade(Loop).LayerType == csVenetianVertical) {
+            if (BITF_TEST_ANY(BITF(state.dataHeatBal->ComplexShade(Loop).LayerType),
```

Comment:
I still have questions about using these macros vs using constexpr functions, but they can wait for another time.  And I need to go back and redigest the discussion from when they were discussed previously before I make any further commentary.